spec/features/action_take_spec.rb

```diff
@@ -0,0 +1,42 @@
+
+# Tests if names are submitted
+feature 'Taking Action' do
+
+  scenario '#Waiting Room' do
+    visit('/')
+
+    # Fill in the form
+    sign_in_and_play
+
+    click_button 'p1-rock'
+
+    expect(page).to have_content "Rock"
+  end
+
+  scenario '#Both Actions' do
+    visit('/')
+
+    # Fill in the form
+    sign_in_and_play
```

Comment:
Wonder whether the visit '/' can be included in the sign_in_and_play web helper? It's such a small change and I wonder if it's actually clearer to leave it separate - I'm struggling to think of any improvements though!